datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/manifest.py

```diff
@@ -44,11 +44,27 @@
     'process_signatures',
 }
 
+METRIC_TO_CHECK_WHITELIST = {
+    'openstack.controller',  # "Artificial" metric, shouldn't be listed in metadata file.
```

Comment:
mmm not sure about this. Could it be done with an annotation on the actual integrations? A comment or something on the manifest similar to the #noqa

Reply:
Can't do comments in JSON unfortunately

Reply:
Yes, unfortunately no way to embed the whitelisting into the manifest file…